common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -537,7 +538,7 @@ public static void beginInteractive(final InteractiveTokenCommand command) {
                     LocalBroadcastManager.getInstance(command.getParameters().getAndroidApplicationContext());
 
             // only send broadcast to cancel if within broker
```

Comment:
looks this comment needs to be updated now

Reply:
Yes.